assets/src/edit-story/components/mediaPicker/useMediaPicker.js

```diff
@@ -91,15 +91,22 @@ export default function useMediaPicker({
     });
 
     // When an image is selected, run a callback.
-    fileFrame.on('select', () => {
+    fileFrame?.once('select', () => {
```

Comment:
Again, why does once and conditionals help here?

Reply:
Just a safety measure to avoid inserting elements & generating posters multiple times should the event be fired more than once.